benefits/eligibility/templates/eligibility/includes/media-item--idcardcheck--start--veteran.html

```diff
@@ -3,24 +3,25 @@
 {% load i18n %}
 
 {% block heading %}
-  {% translate "Access to your preferred VA related account" %}
+    {% translate "Access to your preferred VA related account" %}
 {% endblock heading %}
 
 {% block body %}
-  <div class="media-body--details">
-    <p>
-      {% translate "You will need to sign into VA.gov in order to verify your identity and your veteran status. VA.gov uses the following services:" %}
-    </p>
-    <div class="media-body--items">
-      <ul>
-        <li>{% translate "Login.gov" %}</li>
-        <li>{% translate "ID.me" %}</li>
-        <li>{% translate "DS Logon" %}</li>
-        <li>{% translate "My HealtheVet" %}</li>
-      </ul>
-      <p class="pt-1">
-        {% translate "If you do not have an account with any of these services, you will need to create one. We recommend using Login.gov." %}
-      </p>
+    <div class="media-body--details">
+        <p>
+            {% translate "You will be able to create an account using your email address if you do not already have one. We use your Login.gov account to verify your identity." %}
+            {% translate "Learn more about identity verification" as text %}
+            {% include "core/includes/modal-trigger.html" with modal="modal--identity-verification" text=text period=True %}
+        </p>
+        <div class="media-body--items">
```

Comment:
While working on this PR, I realized that some `.html` files are set to `Spaces: 2` and some are set to `Spaces: 4`. This is an instance where the `media-item--idcardcheck--start--senior.html` is set to 4 spaces, and the `veteran` file is set to 2 spaces.

In my next PR, I'm comparing 2 files: `selection-label--senior.html` and `selection-label--veteran.html` and have a similar issue: senior is set to 4 spaces, and veteran is 2. 

Not sure why we have this discrepancy across files.

Reply:
I think I've noticed this a few places too, I normally just fix it. All the HTML should be 2 (the CSS is already 2) and we don't have any JS to worry about. 

I think it can happen when new files are created, maybe there's a VS Code setting we can add to help avoid in the future.